main.nf

```diff
@@ -32,6 +32,14 @@ workflow {
 
     batch_effect(imputed_mynorm, params.sample_sheet)
 
+    def n_cpgs
+    if(!params.n_cpgs) {
```

Comment:
`params.n_cpgs` should simply have default value on the top of file, that will be replacable if needed by providing additionall arg to cli